src/usr/local/buildpack/tools/git-lfs.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+require_root
+check_semver "$TOOL_VERSION"
+
+if [[ ! "${MAJOR}" || ! "${MINOR}" || ! "${PATCH}" ]]; then
+  echo "Invalid version: ${TOOL_VERSION}"
+  exit 1
+fi
+
+if ! [ "$(command -v git-lfs)" ]; then
+  echo "Skipping, already installed"
+  exit 0
+fi
+
+LFS_FILE="git-lfs-linux-amd64-${TOOL_VERSION}.tar.gz"
```

Comment:
Make `ARCH` configurable like in 

https://github.com/containerbase/buildpack/blob/52800f94727f582808d8bc4247f380076554b769/src/usr/local/buildpack/tools/php.sh#L29
https://github.com/containerbase/buildpack/blob/52800f94727f582808d8bc4247f380076554b769/src/usr/local/buildpack/tools/php.sh#L56